reference/java-api.md

```diff
@@ -669,30 +669,30 @@ The following tables describe the Java classes and their methods.
-| &nbsp;&nbsp; public double [getMinPosition](servo.md#wb_servo_set_position)();                                  |
-| &nbsp;&nbsp; public double [getMaxPosition](servo.md#wb_servo_set_position)();                                  |
-| &nbsp;&nbsp; public void [enablePosition](servo.md#wb_servo_enable_position)(int ms);                           |
-| &nbsp;&nbsp; public void [disablePosition](servo.md#wb_servo_enable_position)();                                |
-| &nbsp;&nbsp; public int [getPositionSamplingPeriod](servo.md#wb_servo_enable_position)();                       |
-| &nbsp;&nbsp; public double [getPosition](servo.md#wb_servo_enable_position)();                                  |
-| &nbsp;&nbsp; public void [enableMotorForceFeedback](servo.md#wb_servo_enable_motor_force_feedback)(int ms);     |
-| &nbsp;&nbsp; public void [disableMotorForceFeedback](servo.md#wb_servo_enable_motor_force_feedback)();          |
-| &nbsp;&nbsp; public int [getMotorForceFeedbackSamplingPeriod](servo.md#wb_servo_enable_motor_force_feedback)(); |
-| &nbsp;&nbsp; public double [getMotorForceFeedback](servo.md#wb_servo_enable_motor_force_feedback)();            |
-| &nbsp;&nbsp; public void [setForce](servo.md#wb_servo_set_force)(double force);                                 |
-| &nbsp;&nbsp; public int [getType](servo.md#wb_servo_get_type)();                                                |
-| }                                                                                                               |
+|                                                                                                                          |
+| ------------------------------------------------------------------------------------------------------------------------ |
+| import com.cyberbotics.webots.controller.Servo;                                                                          |
+| public class [Servo](servo.md) extends [Device](#java_device) {                                                          |
+| &nbsp;&nbsp; public final static int ROTATIONAL, LINEAR;                                                                 |
+| &nbsp;&nbsp; public void [setPosition](servo.md#wb_servo_set_position)(double position);                                 |
+| &nbsp;&nbsp; public double [getTargetPosition](servo.md#wb_servo_set_position)();                                        |
+| &nbsp;&nbsp; public void [setVelocity](servo.md#wb_servo_set_position)(double vel);                                      |
+| &nbsp;&nbsp; public void [setAcceleration](servo.md#wb_servo_set_position)(double force);                                |
+| &nbsp;&nbsp; public void [setMotorForce](servo.md#wb_servo_set_position)(double motor\_force);                           |
+| &nbsp;&nbsp; public double [getMinPosition](servo.md#wb_servo_set_position)();                                           |
```

Comment:
why did you change the order of the functions?
I would keep `getMiniPosition` and `getMaxPosition` together.